valid1.js

```diff
@@ -0,0 +1,61 @@
+const testEnv = '  DATABASE_URL\nADMIN_EMAIL  DEBUG';
+let envValues = []; //save env fields
+let tempStringToParse = testEnv; //string for parsing
+let regExpA = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_';  //valid chars
+
+console.log(' TURNING \n');
+console.log(testEnv);
+console.log('\n  TO  \n');
+
+//PARSING
+for(let i = 0;i<=tempStringToParse.length;i++){ //until the string ends
```

Comment:
We can drastically simplify this 😄.

JavaScript lets us split a string into an array of multiple strings using the [`split` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) in strings.

So we can split our string on `\n` and work elegantly with every line. Example:

```javascript
const multilineString = 'a\nb\nc\nd';
const lines = multilineString.split('\n');

lines.forEach(line => console.log('This is a nice line!', line));
```